config-templates/google/gcp-compute-persistent-disk-csi-driver/1.7.1/deployment.yaml

```diff
@@ -0,0 +1,832 @@
+      kind: ClusterRole
+      name: csi-gce-pd-provisioner-role
+    subjects:
+    - kind: ServiceAccount
+      name: csi-gce-pd-controller-sa
+      namespace: gce-pd-csi-driver
+  - apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+      name: csi-gce-pd-controller-snapshotter-binding
+      labels:
+        app.kubernetes.io/name: gcp-compute-persistent-disk-csi-driver
+        app.kubernetes.io/version: 1.7.1
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: csi-gce-pd-snapshotter-role
+    subjects:
+    - kind: ServiceAccount
+      name: csi-gce-pd-controller-sa
+      namespace: gce-pd-csi-driver
+  - apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+      name: csi-gce-pd-node
+      labels:
+        app.kubernetes.io/name: gcp-compute-persistent-disk-csi-driver
+        app.kubernetes.io/version: 1.7.1
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: csi-gce-pd-node-deploy
+    subjects:
+    - kind: ServiceAccount
+      name: csi-gce-pd-node-sa
+      namespace: gce-pd-csi-driver
+  - apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+      name: csi-gce-pd-resizer-binding
+      labels:
+        app.kubernetes.io/name: gcp-compute-persistent-disk-csi-driver
+        app.kubernetes.io/version: 1.7.1
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: csi-gce-pd-resizer-role
+    subjects:
+    - kind: ServiceAccount
+      name: csi-gce-pd-controller-sa
+      namespace: gce-pd-csi-driver
+  - apiVersion: scheduling.k8s.io/v1
+    description: This priority class should be used for the GCE PD CSI driver controller
+      deployment only.
+    globalDefault: false
+    kind: PriorityClass
+    metadata:
+      name: csi-gce-pd-controller
+      labels:
+        app.kubernetes.io/name: gcp-compute-persistent-disk-csi-driver
+        app.kubernetes.io/version: 1.7.1
+    value: 900000000
+  - apiVersion: scheduling.k8s.io/v1
+    description: This priority class should be used for the GCE PD CSI driver node deployment
+      only.
+    globalDefault: false
+    kind: PriorityClass
+    metadata:
+      name: csi-gce-pd-node
+      labels:
+        app.kubernetes.io/name: gcp-compute-persistent-disk-csi-driver
+        app.kubernetes.io/version: 1.7.1
+    value: 900001000
+  - apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: csi-gce-pd-controller
+      namespace: gce-pd-csi-driver
+      labels:
+        app.kubernetes.io/name: gcp-compute-persistent-disk-csi-driver
+        app.kubernetes.io/version: 1.7.1
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          app: gcp-compute-persistent-disk-csi-driver
+      template:
+        metadata:
+          labels:
+            app: gcp-compute-persistent-disk-csi-driver
```

Comment:
lets add both labels here so that selector will find the pods
```
app.kubernetes.io/name: gcp-compute-persistent-disk-csi-driver
app.kubernetes.io/version: 1.7.1
```